cloudify_cli/config/parser_config.py

```diff
@@ -312,18 +315,17 @@ def parser_config():
                                 .format(DEFAULT_BLUEPRINT_FILE_NAME)
                         },
                     '-d,--deployment-id': deployment_id_argument(
-                            hlp='The id of the deployed blueprint'
+                            hlp='A user provided deployment ID'
                         ),
                     '-i,--inputs':
-                        inputs_argument('Inputs file/string for the deployment'
-                                        ' creation ({0}). '
+                        inputs_argument('Inputs for the deployment ({0}). '
                                         'This argument can be used multiple times. '
                                         '(default: {1})'
                                         .format(FORMAT_INPUT_AS_YAML_OR_DICT,
                                                 DEFAULT_INPUTS_PATH_FOR_INSTALL_COMMAND)
                                         ),
                     '-w,--workflow': make_optional(workflow_id_argument(
-                            hlp='The workflow to start '
+                            hlp='A workflow name to execute '
```

Comment:
It sound a bit like we are executing the workflow's name, but I can't think of a better suggestion.